modules/admin/variables.tf

```diff
@@ -14,10 +14,10 @@
  * limitations under the License.
  */
 
-variable "project_id" {
-  description = "The project ID to deploy to"
-}
-
-variable "bucket_name" {
-  description = "The name of the bucket to create"
-}
+variable "project_id" { default = "" }
```

Comment:
We should never have empty/non-usable default values.

If the value is required (like `project_id` is), you should simply not provide a default at all.

Reply:
Not applicable here per se, but an empty value as default for things like "network" or "subnetwork" default to the default network which is a pretty sane default.

Reply:
In those cases, we could use "default" as the default value.

Reply:
removed empty default values